SteamBotUnitTest/SteamTrade/GenericInventoryTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using SteamKit2;
+using SteamTrade;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SteamBotUnitTest.SteamTrade
+{
+    [TestFixture]
+    class GenericInventoryTest
+    {
+        [Test]
+        public void LoadTest()
+        {
+            var inventory = new GenericInventory(new SteamWeb());
+            //76561198104350201 is my friend's Steam account with lots of Dota2 items which will test the start/more way of loading. It's inventory will remain public and contain more than 2000 items long term.
```

Comment:
He meant because the servers could (and do) go down at any random time, causing the automated unit tests to randomly fail _(meaning AppVeyor will give false-negatives on some PRs)_.  You should never use real data in unit tests.